### DISCUSSION TITLE
Searching by specific field (managed property)

### DISCUSSION THREAD

**Discussion by Zerg00s** [microsoft-search/pnp-modern-search#1978] [Ideas]
Search by specific field

**Describe the solution you'd like**
I would like an ability for the search filters webpart to support a plain textbox. 

**Describe alternatives you've considered**
Currently, we can search by specific fields (managed properties) if we use this syntax: Title:value. This works fine, however, users need to be trained to use this syntax. This is also awkward for them to get used to.

**Additional context**

![image](https://user-images.githubusercontent.com/2797648/160669576-91984f4a-ee6b-4cf2-8612-2faab1dfe51b.png)

**Comment by wobba** [comment]
My personal opinion is that advanced search solutions are used by archivers and librarians. For most people it's too advanced and not used a lot. I see the point of teaching to use properties, but in general it's not used a lot. If you want to build an advanced search box or add the functionality to the one we have, we can certainly consider adding it - but for most users it will clutter the search experience more than helping them.

Having filters works a lot better from a UX point of view.

**Reply by Zerg00s** [reply]
I'm not sure if I can do it (It might be too difficult for me to figure out), but would you be open to a PR that includes a simple textbox as an option for filters? There are definitely valid use cases for this. I am facing it right now. I was asked about it by multiple users during a testing phase and I don't have anything to offer them except for using a tricky semi-manual property search.

**Reply by wobba** [reply]
Supporting a text box, and passing the value down as a filter to see if something returns could be a good idea - and we could support this if someone adds it. Don't see why not.

**Comment by clayjam** [comment]
I would also like this feature and it would provide a lot of flexibility and add real value to users. @wobba - is this something your team can implement as the general community doesn't have these type of skills to develop?

**Reply by Zerg00s** [reply]
I spent a few hours trying to figure it out last year, but unfortunately, I had to give up in the end.

**Reply by wobba** [reply]
@clayjam we don't have a team and rely on the community and spare time to implement functionality. My own time these days is very limited to doing development.
